test/e2e/Makefile

```diff
@@ -70,3 +76,10 @@ dev: $(TOOLING_BINARIES) $(E2E_DATA_DIR) push-ccm-image
 
 clean:
 	rm -rf $(E2E_DATA_DIR) $(TMP_CAPV_DIR)
+
+# dev-exist-image is a convenience target to run the E2E tests with the development image already pushed to any registry
+# (e.g. dockerhub). No need push permission of gcr.io/cloud-provider-vsphere/cpi/pr/manager
+dev-exist-image: $(TOOLING_BINARIES) $(E2E_DATA_DIR)
```

Comment:
instead of `dev-exist-image`, maybe `run-e2e-with-dev-image` can explain this command better?

Reply:
Yes, run-e2e-with-dev-image sounds better.